### PR TITLE
Add Wordpress

### PR DESCRIPTION
**Pull request by moan0s**
From #12

### CODE REVIEW

File: docs/services/wordpress.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+mariadb_root_passsword: ''
+
+########################################################################
+#                                                                      #
+# /mariadb                                                             #
+#                                                                      #
+########################################################################
+```
+
+
```
Comment (spantaleev) [inline docs/services/wordpress.md:55]: Isn't this unnecessary? We already list and link to MariaDB (`mariadb.md`) in the **Dependencies** section, so duplicating MariaDB's installation instructions here seems redundant - we don't do this for other services.

File: templates/group_vars_mash_servers [hunk 2]
```diff
@@ -5384,3 +5400,50 @@ devture_woodpecker_ci_agent_config_agent_secret: "{{ devture_woodpecker_ci_serve
...
+wordpress_identifier: "{{ mash_playbook_service_identifier_prefix }}wordpress"
+
+wordpress_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}wordpress"
+
+wordpress_uid: "{{ mash_playbook_uid }}"
+wordpress_gid: "{{ mash_playbook_gid }}"
+
+wordpress_systemd_required_systemd_services_list_auto: |
+  {{
+    ([mariadb_identifier ~ '.service'] if mariadb_enabled and wordpress_database_hostname == mariadb_identifier else [])
```
Comment (spantaleev) [inline templates/group_vars_mash_servers:5423]:
```suggestion
    ([(mariadb_identifier | default('mash-mariadb')) ~ '.service'] if mariadb_enabled | default(false) and wordpress_database_hostname == mariadb_identifier | default('mash-mariadb') else [])
```

File: templates/group_vars_mash_servers [hunk 3]
```diff
@@ -5384,3 +5400,50 @@ devture_woodpecker_ci_agent_config_agent_secret: "{{ devture_woodpecker_ci_serve
...
+  }}
+
+
+wordpress_container_additional_networks_auto: |
+  {{
+    ([mash_playbook_reverse_proxyable_services_additional_network] if mash_playbook_reverse_proxyable_services_additional_network else [])
+    +
+    ([mariadb_identifier] if mariadb_enabled and wordpress_database_hostname == mariadb_identifier and wordpress_container_network != mariadb_identifier else [])
```
Comment (spantaleev) [inline templates/group_vars_mash_servers:5431]:
```suggestion
    ([mariadb_identifier | default('mash-mariadb')] if mariadb_enabled | default(false) and wordpress_database_hostname == mariadb_identifier | default('mash-mariadb') and wordpress_container_network != mariadb_identifier | default('mash-mariadb') else [])
```

File: templates/group_vars_mash_servers [hunk 4]
```diff
@@ -5384,3 +5400,50 @@ devture_woodpecker_ci_agent_config_agent_secret: "{{ devture_woodpecker_ci_serve
...
+  }}
+
+wordpress_container_labels_traefik_enabled: "{{ mash_playbook_traefik_labels_enabled }}"
+wordpress_container_labels_traefik_docker_network: "{{ mash_playbook_reverse_proxyable_services_additional_network }}"
+wordpress_container_labels_traefik_entrypoints: "{{ devture_traefik_entrypoint_primary }}"
+wordpress_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certResolver_primary }}"
+
+
+wordpress_database_hostname: "{{ mariadb_identifier if mariadb_enabled | default(false) else '' }}"
+wordpress_mysql_port: "{{ '3306' if mariadb_enabled else '' }}"
```
Comment (spantaleev) [inline templates/group_vars_mash_servers:5441]:
```suggestion
wordpress_mysql_port: "{{ '3306' if mariadb_enabled | default(false) else '' }}"
```

File: templates/group_vars_mash_servers [hunk 5]
```diff
@@ -3930,6 +3935,17 @@ mash_playbook_mariadb_managed_databases_auto_itemized:
...
   # /role-specific:authelia
 
+  # role-specific:wordpress
+  - |-
+    {{
+      ({
+        'name': wordpress_database_name,
+        'username': wordpress_database_username,
+        'password': wordpress_database_password,
+      } if wordpress_enabled and wordpress_database_hostname == mariadb_identifier else omit)
```
Comment (spantaleev) [inline templates/group_vars_mash_servers:3945]:
```suggestion
      } if wordpress_enabled and wordpress_database_hostname == mariadb_identifier | default('mash-mariadb') else omit)
```

File: docs/services/wordpress.md (hunk 1 above)
Comment (moan0s) [reply docs/services/wordpress.md:55]: True, fixed

**Pull request merged**